Statistics/Statistics.cs

```diff
@@ -1,12 +1,28 @@
-﻿using System;
+﻿  
+using System;
 using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
 
 namespace Statistics
 {
+    
     public class StatsComputer
     {
-        public Stats CalculateStatistics(List<float> numbers) {
-            //Implement statistics here
+        public Stats CalculateStatistics(List<double> numbers)
+        {
+            Stats obj = new Stats();
```

Comment:
All modifications are good 👍 
Give attention to the variable names. The obj doesn't convey the meaning of the variable.